feeding/src/FeedingDemo.cpp

```diff
@@ -182,8 +183,18 @@ FeedingDemo::FeedingDemo(
   mTiltOffset = Eigen::Vector3d(
       tiltOffsetVector[0], tiltOffsetVector[1], tiltOffsetVector[2]);
 
-  mVelocityLimits
+  std::vector<double> velocityLimits
       = getRosParam<std::vector<double>>("/study/velocityLimits", *mNodeHandle);
+  mVelocityLimits = DEFAULT_VELOCITY_LIM * Eigen::Vector6d::Ones();
+
+  // If /study/velocityLimits size < 6
```

Comment:
Nit: Is there a reason these comment lines are shorter than the standard 80 char limit?